doc/man3/SSL_alert_type_string.pod

```diff
@@ -36,7 +36,7 @@ and does not influence the normal data stream (unless its contents results
 in the communication being canceled).
 
 A warning alert is sent, when a non-fatal error condition occurs. The
-"close notify" alert is sent as a warning alert. Other examples for
+close_notify alert is sent as a warning alert. Other examples for
 non-fatal errors are certificate errors ("certificate expired",
```

Comment:
Hmmm. I see only now that all the other errors are listed in double quotes here. It does not make sense to change "close notify" only, without also changing "certificate expired", "unsupported certificate", etc.  And since the documented return values can not be changed, it is probably better to leave everything as it is. So just drop the fixup if you like. Sorry for the inconvenience!

Reply:
Thinking once more about it, I think the best solution is to revert the renaming in the file `SSL_alert_type_string.pod` (and only there). Because in that file the documentation talks about the returned string values of `SSL_alert_desc_string_long()`, and `"close notify"` is one of them.

Reply:
But I leave the decision up to you.